src/Session.php

```diff
@@ -994,16 +994,24 @@ public static function checkValidSessionId()
         } else {
             $user_table = User::getTable();
             $pu_table   = Profile_User::getTable();
+            $profile_table = Profile::getTable();
             $result = $DB->request(
                 [
                     'COUNT'     => 'count',
+                    'SELECT'    => [$profile_table . '.last_rights_update'],
```

Comment:
IMHO, if you add this non aggregated field to selection, you should add a `'GROUPBY' => [$profile_table . 'id']`.